overview/demo/shell.nix

```diff
@@ -32,7 +32,31 @@ let
         + ''
           export PS1="\[\033[1m\][demo-shell]\[\033[m\]\040\w >\040"
 
-          ${pkgs.lib.getExe pkgs.bash} --norc "$@"
+          ${lib.optionalString (demo-shell != { }) ''
+            echo -e "\n\033[1;32mDemo shell activated! Available programs:\033[0m"
+            ${lib.concatStringsSep "\n" (
+              lib.mapAttrsToList (
```

Comment:
Also, we already have the programs in `runtimeInputs`, so we can just iterate over that.